utility/src/main/java/org/oppia/util/logging/firebase/FirebaseEventLogger.kt

```diff
@@ -22,3 +24,12 @@ class FirebaseEventLogger(
     firebaseAnalytics.logEvent(eventLog.actionName.toString(), bundle)
   }
 }
+
+/** Temporary logger class that performs no action when called upon.*/
+class TempEventLogger : EventLogger, ExceptionLogger {
```

Comment:
Let's call this StubbedEventLogger instead

Reply:
Done.